k8s/src/test/scala/io/buoyant/k8s/IngressCacheTest.scala

```diff
@@ -1,12 +1,228 @@
 package io.buoyant.k8s
 
+import com.twitter.finagle.Service
+import com.twitter.io.Buf
+import com.twitter.util.Future
 import io.buoyant.test.Awaits
 import org.scalatest.FunSuite
+import com.twitter.finagle.http.{Request => FRequest, Response => FResponse}
```

Comment:
Indeed. I'm happy with keeping the name, but something for consideration: Because there are so many types called things like `Request`, `Response`, `Logger`, and `Service` in the classpath, at SoundCloud we had a convention that every time we imported one of those we'd rename it so that we could identify at a glance where it came from. e.g. we'd always do

```scala
import com.twitter.finagle.http.{Request => FFInagleRequest, Response => FInagleResponse}
import com.soundcloud.ourinternalstuff.http.{Request => ScRequest, Response => ScResponse}
```

But if we don't follow anything like this convention I'm happy to simplify.